src/main/java/uk/gov/hmcts/reform/sscs/helpers/CaseFlagMappingServiceHelper.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.sscs.helpers;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
```

Comment:
This annotation shouldn't be used if class is manually instantiated

Reply:
Yes the class was meant to be un instantiated sorry..